Platform/IdavilleBoardPkg/Library/AspeedGfxLib/ast_mode.c

```diff
@@ -575,8 +575,10 @@ static void ast_crtc_dpms(struct drm_crtc *crtc, int mode)
         break;
     case DRM_MODE_DPMS_STANDBY:
         ch = 0x01;
+        break;
     case DRM_MODE_DPMS_SUSPEND:
         ch = 0x02;
+        break;
```

Comment:
From the code logic, we should not add "break" here since code for these 3 cases shared same logic to program "ch"
Please double check it.
You could refer latest code https://github.com/torvalds/linux/blob/5d0c230f1de8c7515b6567d9afba1f196fb4e2f4/drivers/gpu/drm/ast/ast_mode.c#L1049C3-L1049C6

Reply:
Updated accordingly